### DISCUSSION TITLE
How link one record from table A to another record from table B via RestApi?

### DISCUSSION THREAD

**Discussion by tusxxx** [nocodb/nocodb#6771] [Q&A]
OneToMany

**Comment by pedralmeida22** [comment]
Source: https://apis.nocodb.com/#tag/DB-Table-Row/operation/db-table-row-nested-add

POST http://localhost:8080/api/v1/db/data/{orgs}/{baseName}/{tableName}/{rowId}/{relationType}/{columnName}/{refRowId}
